library/Fission/Models.hs

```diff
@@ -121,6 +124,8 @@ App
   ownerId     UserId
   cid         CID
 
+  size        Natural
```

Comment:
And here. For example, is this the tree size (number of links) or the size on disk (bytes). A newtype wrapper can help!